catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/QueryMetacardTypeImpl.java

```diff
@@ -126,6 +128,15 @@
             false /* tokenized */,
             false /* multivalued */,
             BasicTypes.STRING_TYPE));
+
+    DESCRIPTORS.add(
+        new AttributeDescriptorImpl(
+            QUERY_IS_ADVANCED,
```

Comment:
❓ Have we done anything to check if this impacts backwards compatibility with old workspaces?

Reply:
It won't have any impact.  Old workspace queries will default to false for isAdvanced, so they will act as they would have previously (basic view, with the user having to know to switch to advanced if necessary).